src/gql/queries/property.ts

```diff
@@ -81,22 +77,24 @@ export const CHANGE_PROPERTY_VALUE = gql`
   ${layerFragment}
 `;
 
-export const UPLOAD_FILE_TO_PROPERTY = gql`
-  mutation UploadFileToProperty(
+export const CHANGE_PROPERTY_VALUE = gql`
+  mutation ChangePropertyValue(
```

Comment:
Can you change these to match the backend. "UpdateXXXXX" is used many other places, and (correct me if I am wrong) I don't think we have a "ChangeXXXX" anywhere else

Reply:
yes you're right there's no 'ChangeXXX' anywhere else. sorry didn't correct that earlier